cypress/features/regression/designSystem/grid.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Design System Grid component
+  I want to test Design System Grid component
+
+  Background: Open Design System Grid component page
+    Given I open Design Systems basic "Grid" component docs page
+
+  @positive
+  Scenario: I confirm justify style
+    # When I click on component
+    Then justify pod 0 has style for justify left
+      And justify pod 1 has style for justify center
+      And justify pod 2 has style for justify right
+
+  @positive
+  Scenario: I confirm align style
+    # When I click on component
```

Comment:
Uncomment this line please, if it is necessary

Reply:
fixed